### ISSUE TITLE
Rustic Redstone Semaphore crashing client

### ISSUE THREAD

**Issue by Aquabrodie** [AllTheMods/ATM-5#173]
## Expected Behavior
Being able to be placed in the world normally

## Current Behavior
crashing when placed

## Steps to Reproduce
1. Craft Rustic Redstone Semaphore
2. Place Rustic Redstone Semphore
3. Crash

## Log
https://gist.github.com/Aquabrodie/5dd378d534d8762c2d1b863784654119

## Detailed Description
To be honest I'm not really sure what the block does, its why I was trying to place it and figure it out, the description suggests something with flags. **I will mention that this was on a server so I'm not sure if the same can be said for single player.**

**Comment by Hanhula** [comment]
I'll note that I'm the server admin here, and got this in console around when he crashed:

```
19.06 07:06:41 [Server] Server thread/FATAL Error executing task on Server
19.06 07:06:41 [Server] INFO java.lang.IllegalArgumentException: Cannot set property IntegerProperty{name=power, clazz=class java.lang.Integer, values=[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]} as it does not exist in Block{rsgauges:rustic_semaphore}
19.06 07:06:41 [Server] INFO at net.minecraft.state.StateHolder.func_206870_a(SourceFile:104) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
19.06 07:06:41 [Server] INFO at wile.rsgauges.blocks.BlockGauge.func_196258_a(BlockGauge.java:131) ~[?:1.2.4-b1] {re:classloading}
19.06 07:06:41 [Server] INFO at net.minecraft.item.BlockItem.func_195945_b(BlockItem.java:102) ~[?:?] {re:classloading,xf:fml:quark:alter-placement-state,pl:runtimedistcleaner:A}
19.06 07:06:41 [Server] INFO at net.minecraft.item.BlockItem.func_195942_a(BlockItem.java:51) ~[?:?] {re:classloading,xf:fml:quark:alter-placement-state,pl:runtimedistcleaner:A}
19.06 07:06:41 [Server] INFO at net.minecraft.item.BlockItem.func_195939_a(BlockItem.java:39) ~[?:?] {re:classloading,xf:fml:quark:alter-placement-state,pl:runtimedistcleaner:A}
19.06 07:06:41 [Server] INFO at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:610) ~[?:?] {re:mixin,re:classloading}
19.06 07:06:41 [Server] INFO at shadows.apotheosis.deadly.asm.DeadlyHooks.onItemUse(DeadlyHooks.java:120) ~[?:3.1.2] {re:classloading}
19.06 07:06:41 [Server] INFO at net.minecraft.item.ItemStack.func_196084_a(ItemStack.java) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,xf:fml:apotheosis:apothitemusehook,xf:fml:quark:add-spacing-for-stack,pl:runtimedistcleaner:A}
19.06 07:06:41 [Server] INFO at net.minecraft.server.management.PlayerInteractionManager.func_219441_a(PlayerInteractionManager.java:348) ~[?:?] {re:classloading,pl:accesstransformer:B}
19.06 07:06:41 [Server] INFO at net.minecraft.network.play.ServerPlayNetHandler.func_184337_a(ServerPlayNetHandler.java:853) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,xf:fml:randompatches:RandomPatches ServerPlayNetHandler Transformer}
19.06 07:06:41 [Server] INFO at net.minecraft.network.play.client.CPlayerTryUseItemOnBlockPacket.func_148833_a(SourceFile:36) ~[?:?] {re:classloading}
19.06 07:06:41 [Server] INFO at net.minecraft.network.play.client.CPlayerTryUseItemOnBlockPacket.func_148833_a(SourceFile:10) ~[?:?] {re:classloading}
19.06 07:06:41 [Server] INFO at net.minecraft.network.PacketThreadUtil.func_225383_a(SourceFile:21) ~[?:?] {re:classloading}
19.06 07:06:41 [Server] INFO at net.minecraft.util.concurrent.TickDelayedTask.run(SourceFile:18) ~[?:?] {re:classloading}
19.06 07:06:41 [Server] INFO at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213166_h(SourceFile:144) [?:?] {re:classloading,pl:accesstransformer:B}
19.06 07:06:41 [Server] INFO at net.minecraft.util.concurrent.RecursiveEventLoop.func_213166_h(SourceFile:23) [?:?] {re:classloading}
19.06 07:06:41 [Server] INFO at net.minecraft.server.MinecraftServer.func_213166_h(MinecraftServer.java:731) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
19.06 07:06:41 [Server] INFO at net.minecraft.server.MinecraftServer.func_213166_h(MinecraftServer.java:141) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
19.06 07:06:41 [Server] INFO at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213168_p(SourceFile:118) [?:?] {re:classloading,pl:accesstransformer:B}
19.06 07:06:41 [Server] INFO at net.minecraft.server.MinecraftServer.func_213205_aW(MinecraftServer.java:714) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
19.06 07:06:41 [Server] INFO at net.minecraft.server.MinecraftServer.func_213168_p(MinecraftServer.java:708) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
19.06 07:06:41 [Server] INFO at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213161_c(SourceFile:127) [?:?] {re:classloading,pl:accesstransformer:B}
19.06 07:06:41 [Server] INFO at net.minecraft.server.MinecraftServer.func_213202_o(MinecraftServer.java:694) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
19.06 07:06:41 [Server] INFO at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:641) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
19.06 07:06:41 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_241] {}
19.06 07:06:56 [Disconnect] User aquabrodie has disconnected, reason: Disconnected
```

**Comment by stfwi** [comment]
Hi guys, that's my mistake (placement default state bug). I pushed the fix on Curse (version `rsgauges-1.15.2-1.2.4-b2`). 
So, @ATMTeam, if you could include this mod version in the next pack update, would be cool.
Cheers, wile -

**Comment by whatthedrunk** [comment]
Updated in 2.5 that will be out later today thanks!
